auraescript/src/builtin/auraescript_client.rs

```diff
@@ -0,0 +1,82 @@
+use anyhow::Result;
+use client::{Client, AuraeConfig};
+use deno_core::{OpState, Resource, ResourceId};
+use std::{cell::RefCell, rc::Rc};
+
+// `AuraeConfig` `try_default`
+#[deno_core::op]
+pub(crate) async fn as__aurae__config__try_default(
```

Comment:
I'm trying to understand the naming convention here. I see you used  "as" instead of "ae" (makes sense since these ops are auraescript specific), but I'm not sure what the logic for the middle is. I would think this would be `as__builtin__client__try_default` and that the others would follow the same pattern.

Reply:
I choose aurae_config because try_default is a `AuraeConfig` method

Reply:
Ah, that makes sense. So this is nit: there is an extra "_" between `aurae` and `config`, IMO ( `as__aurae_config__try_default`)